logbook-api/src/main/java/org/zalando/logbook/attributes/AllAttributesExtractor.java

```diff
@@ -0,0 +1,28 @@
+package org.zalando.logbook.attributes;
+
+import org.apiguardian.api.API;
+import org.zalando.logbook.HttpRequest;
+import org.zalando.logbook.HttpResponse;
+
+import javax.annotation.Nonnull;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+@API(status = EXPERIMENTAL)
+public class AllAttributesExtractor implements AttributeExtractor {
```

Comment:
I conceived the `AllAttributesExtractor` class as an alternative to `NoOpAttributeExtractor`. I see this as a frequent use case. By getting such a class from the library, users won't have to implement it themselves if they need to get all attributes.

As for the location of the class, I need your advice as you have more experience with the project.

Reply:
I see your reasoning.

I suggest moving the class and it's test to `logbook-core` module, package `org.zalando.logbook.core.attributes`.
There are already some implementations `AttributeExtractor` there, and this can be another useful implementation to that collection.

You can also make the class `final`.